pkg/storage/driver/driver.go

```diff
@@ -160,6 +160,26 @@ func InitMockDB() {
 	storage.InitStores(db)
 }
 
+func InitMockCache() {
```

Comment:
It might be better to use `InitCache` when run unit test.

Reply:
ok